README.md

```diff
@@ -477,6 +477,20 @@ colorful_winsep = {
 </tr>
 <!-- colorful_winsep.nvim -->
 
+<!-- dadbod-ui -->
```

Comment:
Change to `vim-dadbob-ui` to show its real name, as well as line 483 and 492, but no need to change the integration name in line 487. Also remember to move this chunk down to integrations start with `v`.